tests/test_workflow.py

```diff
@@ -87,9 +87,10 @@ def test_submit(self, api: V1alpha1Api, wf: Workflow) -> None:
         wf.spec.arguments = V1alpha1Arguments(
             parameters=[V1alpha1Parameter(name="param")]
         )
-        workflow_name: str = wf.submit(
+        workflow_result: str = wf.submit(
             client=api, namespace="test", parameters={"param": "test"}
         )
 
-        assert isinstance(workflow_name, str)
-        assert workflow_name == "test"
+        assert isinstance(workflow_result, V1alpha1Workflow)
+        assert isinstance(workflow_result.metadata.name, str)
+        assert workflow_result.metadata.name == "test"
```

Comment:
As this is now a Workflow, we can check that the parameters are present and that its value indeed is `test` :grin: